### PR TITLE
Update Issue and Pull Request Templates Reference Issue #25

### PR DESCRIPTION
**Pull request by Jacksole**
Hey Team

I added a docker file to my repo but I don't think we are ready for that yet. So you can just remove that file change.

### CODE REVIEW

**Comment by Veradux** [comment]
Is this a PR for both Docker and issue templates? Sounds like it should be in two different PRs.

**Comment by Veradux** [comment]
> Docker file can be deleted as it's not for this pull request

Well then.. delete it. :D Also this seems like a generic issue template. Do you think it would be better to make it specific for the project? The project people have agreed on is a console desktop game. Meaning, these steps are not a fitting example: 
> Go to '...'
Click on '....'
Scroll down to '....'
See error

> Browser [e.g. chrome, safari]

Browser info is also irrelevant. So is the whole smartphone paragraph.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```
Comment (Veradux) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Again, there is no clicking and scrolling to be done in a console game. The only way to interact with it is through text commands.

**Comment by Plat251** [comment]
This can easily be put in Wiki section. It doesn't have to a part of actual codebase.

**Comment by blackk100** [comment]
It may not be used by the project but should be included to allow default templates to be shown by GitHub on the create a new issue form. See: https://help.github.com/en/articles/creating-a-default-community-health-file-for-your-organization and https://help.github.com/en/articles/about-issue-and-pull-request-templates.

**Comment by anthonyLuttrell** [comment]
I'm kind of confused after browsing through all of the commits (trying to get a better understanding for things in general). What is the point of this? I see it says 2 files changes, which appear to be a bug report and feature request template. Is that the only thing happening with this PR?

**Comment by Plat251** [comment]
Yes. This PR creates two template files and nothing else.

**Pull request merged**